javascript-constructors/new-objects.js

```diff
@@ -0,0 +1,23 @@
+var anObject = new Object();
+console.log('value of anObject: ', anObject);
+console.log('typeof anObject: ', anObject);
+
+var anArray = new Array();
+console.log('value of anArray: ', anArray);
+console.log('typeof anArray: ', anArray);
+
+var aFunction = new Function();
+console.log('value of aFunction: ', aFunction);
+console.log('typeof aFunction: ', aFunction);
+
+var aString = new String();
+console.log('value of aString: ', aString);
+console.log('typeof aString: ', aString);
+
+var aNumber = new Number();
+console.log('value of aNumber: ', aNumber);
+console.log('typeof aNumber: ', aNumber);
+
+var aBoolean = new Boolean();
+console.log('value of aBoolean: ', aBoolean);
+console.log('typeof aBoolean: ', aBoolean);
```

Comment:
Please output the type of each value.

Reply:
![javascript-contructors-new-objects-2](https://user-images.githubusercontent.com/90303824/168723061-3b0f95cc-335a-4835-8010-025938712773.png)